projects/js-packages/charts/changelog/charts-141-remove-consumer-product-themes

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: removed
```

Comment:
Should this be a major/breaking change for consumers that might be using these themes?

Reply:
AFAIK we are still not at 1.0 and so breaking changes are allowed, correct @kangzj?

> Just checking if you are planning to also update the [theme in Calypso](https://github.com/Automattic/wp-calypso/blob/424694a051da7a0689b6a1be994ddf87b13e1965/client/my-sites/stats/components/line-chart/index.tsx#L4)?

Yes, we should, thanks for the reminder :)

Reply:
I've checked Calypso, and the default theme and Jetpack theme are so similar (the only difference is the leaderboard chart colors which isn't used) that there will be no difference switching to default. We could just switch, but it's probably worth upgrading the lib dependency while we're at it.

Reply:
Calypso PR ready: https://github.com/Automattic/wp-calypso/pull/107591